Source/DafnyCore/Compilers/Compiler-go.cs

```diff
@@ -3491,7 +3505,7 @@ private class SpecialNativeType : UserDefinedType {
       } else {
         Contract.Assert(ct is SeqType);  // follows from precondition
         if (ct.Arg.IsCharType) {
-          wr.Write("_dafny.SeqOfChars");
+          wr.Write(UnicodeChars ? "_dafny.SeqOfCodePoints" : "_dafny.SeqOfChars");
```

Comment:
What does this branch correspond to?  Do we really need the `SeqOfCodePoints` or could we just fall through to `SeqOf` below in the unicode case?

Reply:
Oh good call, `SeqOfCodePoints` is indeed pointless. :) Removed from the runtime too.